lib/public/app.css

```diff
@@ -316,6 +316,10 @@ label {
     cursor: default;
 }
 
+.not-allowed {
+    cursor: not-allowed;
+}
+
```

Comment:
Please, let's discuss when you want to implement new stuff, including UX.
Why do you want to put it as `not-allowed`? It's simply not a link.

Reply:
it's not a link, but user doesn't know that, for user it's disabled link, therefore it's grey and I used cursor which suggests, that it's normally link, but it's disabled because there is no associated data

Reply:
![image](https://github.com/AliceO2Group/Bookkeeping/assets/31805063/7f4b5d5c-5db7-4c67-acfc-b461a89fef0f)
If you leave it as simple text, it is pretty obvious that it's not links. If you concatenate all, you may need to separate it with a `,` or a `-`, and maybe display `No Runs` or `No Data Pass` when you have 0 items 
Actually, why do you concatenate all of this in one column, not only one? Displaying simply the amount of it?

Reply:
applied